src/inlineSuggestions/snippets/snippetDecoration.ts

```diff
@@ -95,8 +95,11 @@ export function handleClearSnippetDecoration(): void {
 
 function calculateStartAfterUserInput(range: Range): Range | undefined {
   const currentPosition = window.activeTextEditor?.selection.active;
+  const textInsideSnippetBlankRange = window.activeTextEditor?.document.getText(
+    range
+  );
 
-  if (currentPosition) {
+  if (currentPosition && textInsideSnippetBlankRange?.trim() !== "") {
```

Comment:
`textInsideSnippetBlankRange?.trim() !== ""` is not so clear to me, maybe extract it to a named parameter